car_description/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model
+   href="http://download.ros.org/schema/package_format3.xsd"
+   schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>car_description</name>
+    <version>1.0.0</version>
+    <description>The car_description package</description>
+    <maintainer email="alex.bordallo@ed.ac.uk">Alejandro Bordallo</maintainer>
+    <license>Apache-2.0</license>
+
+    <buildtool_depend>ament_cmake</buildtool_depend>
+
+    <depend>xacro</depend>
+    <depend>robot_state_publisher</depend>
+    <depend>joint_state_publisher</depend>
+
+    <test_depend>ament_lint_auto</test_depend>
+    <test_depend>ament_lint_common</test_depend>
```

Comment:
you don't need these unless you use them in the cmake

more on that...
it is really up to you but I'd recommend not recommend ament auto linters. Try pre-commit which you can integrate with any formatting tool, github workflows that run super fast (a few seconds, as it is not a part of your TEST BUILD) and generally keep your testing separate from linting.

https://github.com/ros-controls/ros2_control/blob/master/.pre-commit-config.yaml
https://github.com/ros-controls/ros2_control_ci/blob/master/.github/workflows/ci-pre-commit.yml

Reply:
Thanks for the feedback!
Removed these couple of lines and brought in the pre-commit config file you pointed to. I ran it and (Apart from over-zealously "correcting" a "spelling-mistake" in the .dae file) did well!
I'll play with it for a bit before I tune it further.